### ISSUE TITLE
is it supports native aot?

### ISSUE THREAD

**Issue by ltbyun** [HangfireIO/Hangfire#2478]
when i use hangfire in asp .net core min aot api, it says:

3>Hangfire.NetCore.dll: Warning IL2104 : Assembly 'Hangfire.NetCore' produced trim warnings. For more information see https://aka.ms/dotnet-illink/libraries

**Comment by odinserj** [comment]
No, Native AOT is not supported yet, and requires a huge amount of work to be supported. So, while such support is planned, it will take considerable amount of time to be completed.
